apps/web/styles/components/UtilityStyles.ts

```diff
@@ -0,0 +1,40 @@
+import { styled } from "styled-components";
+
+export const Container = styled.div`
+  display: flex;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  right: 0;
+  padding: 0.1em 1.7em;
+  border-radius: 10px;
+`;
+
+export const ContentRegistration = styled.div`
+  display: flex;
+  gap: 1em;
+  padding-left: 30px;
+  border-left: 1px solid ${({ theme }) => theme.COLORS.DEFAULT};
+`;
+
+export const RegisterButton = styled.button`
```

Comment:
@MFelipeSilva please change this button name

Reply:
changed